packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -26,10 +27,10 @@ import injectSpark from '../../modules/redux-spark/inject-spark';
  * ChatWidget Component
  */
 export class ChatWidget extends Component {
-
   constructor(props) {
     super(props);
     this.getActivityList = this.getActivityList.bind(this);
+    this.handleActivityDelete = this.handleActivityDelete.bind(this);
```

Comment:
Yea, this is getting more redundant every time we add handler. Let's add a todo item to move things to autobinder after this week.